test/e2e/framework/test_context.go

```diff
@@ -398,7 +398,7 @@ func RegisterClusterFlags(flags *flag.FlagSet) {
 	// If this becomes true as well, they should be refactored into RegisterCommonFlags.
 	flags.BoolVar(&TestContext.PrepullImages, "prepull-images", false, "If true, prepull images so image pull failures do not cause test failures.")
 	flags.StringVar(&TestContext.Provider, "provider", "", "The name of the Kubernetes provider (gce, gke, local, skeleton (the fallback if not set), etc.)")
-	flags.StringVar(&TestContext.Tooling, "tooling", "", "The tooling in use (kops, gke, etc.)")
+	flags.StringVar(&TestContext.Tooling, "tooling", "kube-up", "The tooling used to create the cluster (kops, gke, kube-up).")
```

Comment:
we should not set this default

Reply:
This PR is being closed out. I needed to check something in the kubetest2 test.

/close